src/tribler-common/tribler_common/tests/test_patch_import.py

```diff
@@ -38,3 +45,4 @@ async def test_mock_import_strict():
 async def test_mock_import_always_raise_exception_on_import():
     with pytest.raises(ImportError):
         import time
+        time.gmtime(0)
```

Comment:
This call is unnecessary for the test itself, but this line prevents removing "unused" `import time` during IDE autoformatting procedure.

Reply:
**Suggestion:** It would be better to leave this comment in the code.

Reply:
Done.